README.md

```diff
@@ -31,165 +31,202 @@
     target="Survived",  # Ground truth variable
     name="Titanic dataset", # Optional
     cat_columns=['Pclass', 'Sex', "SibSp", "Parch", "Embarked"]  # Optional, but is a MUST if available. Inferred automatically if not.
 )
 
-# Wrap your model with Giskard.Model. Check the dedicated doc page: https://docs.giskard.ai/en/latest/guides/wrap_model/index.html
-# you can use any tabular, text or LLM models (PyTorch, HuggingFace, LangChain, etc.),
-# for classification, regression & text generation.
-def prediction_function(df):
-    # The pre-processor can be a pipeline of one-hot encoding, imputer, scaler, etc.
-    preprocessed_df = data_preprocessor(df)
-    return clf.predict_proba(preprocessed_df)
-
+# Wrap your model with Giskard.Model.
 giskard_model = giskard.Model(
-    model=prediction_function,  # A prediction function that encapsulates all the data pre-processing steps and that could be executed with the dataset used by the scan.
+    model=demo_model,  # A prediction function that encapsulates all the data pre-processing steps and that could be executed with the dataset used by the scan.
     model_type="classification",  # Either regression, classification or text_generation.
     name="Titanic model",  # Optional
     classification_labels=clf.classes_,  # Their order MUST be identical to the prediction_function's output order
     feature_names=['PassengerId', 'Pclass', 'Name', 'Sex', 'Age', 'SibSp', 'Parch', 'Fare', 'Embarked'],  # Default: all columns of your dataset
     # classification_threshold=0.5,  # Default: 0.5
 )
 
-# Then apply the scan
-results = giskard.scan(giskard_model, giskard_dataset)
 ```
 
+✨✨✨Then run the scan✨✨✨
+```python
+results = giskard.scan(giskard_model, giskard_dataset)
+```
 Once the scan completes, you can display the results directly in your notebook:
 
 ```python
-display(scan_results)  # in your notebook
+display(scan_results)
 ```
+*Check our wrapping [model](https://docs.giskard.ai/en/latest/guides/wrap_model/index.html) & [dataset](https://docs.giskard.ai/en/latest/guides/wrap_dataset/index.html) docs for more information.*
+## 2. 🪄 Automatically generate a test suite
 
-### Automatically generate a test suite based on the scan results
-
-If the scan found potential issues in your model, you can automatically generate a test suite.
-
-Generating a test suite from your scan results will enable you to:
-- Turn the issues you found into actionable tests that you can directly integrate in your CI/CD pipeline
-- Diagnose your vulnerabilities and debug the issues you found in the scan
+If the scan found potential issues in your model, you can automatically generate a **test suite**.
 
 ```python
 test_suite = scan_results.generate_test_suite("My first test suite")
-
-# You can run the test suite locally to verify that it reproduces the issues
+```
+You can run the test suite locally to verify that it reproduces the issues
+```python
 test_suite.run()
 ```
+# ❓ FAQ
```

Comment:
You can add a "Where can I get more help?" or something similar and link the discord